frontend/src/layouts/DashboardLayout.vue

```diff
@@ -130,22 +219,46 @@ export default {
     setTunneledEmail(email) {
       this.tunneledEmail = email
     },
+    toogleMobilMenu() {
+      // console.log(this.$refs.sideMenuRow.classList.contains('position-absolute'))
+
+      this.$refs.sideMenuRow.classList.toggle('d-none')
+      this.$refs.sideMenuRow.classList.toggle('position-absolute')
+      // console.log(document.getElementById('component-sidebar'))
+      document.getElementById('side-menu').classList.toggle('bg-lightgrey')
+      this.hamburger ? (this.hamburger = false) : (this.hamburger = true)
+    },
+    dark() {
+      document.getElementById('app').classList.add('dark-mode')
+      document.querySelector('#app a').classList.add('dark-mode')
+      this.darkMode = true
+    },
+
+    light() {
+      document.getElementById('app').classList.remove('dark-mode')
+      document.querySelector('#app a').classList.remove('dark-mode')
+      this.darkMode = false
+    },
+
+    modeToggle() {
+      if (this.darkMode || document.getElementById('app').classList.contains('dark-mode')) {
+        this.light()
+      } else {
+        this.dark()
+      }
+    },
   },
   computed: {
-    elopageUri() {
-      const pId = this.$store.state.publisherId
-        ? this.$store.state.publisherId
-        : CONFIG.DEFAULT_PUBLISHER_ID
-      return encodeURI(
-        this.$store.state.hasElopage
-          ? `https://elopage.com/s/gradido/sign_in?locale=${this.$i18n.locale}`
-          : `https://elopage.com/s/gradido/basic-de/payment?locale=${this.$i18n.locale}&prid=111&pid=${pId}&firstName=${this.$store.state.firstName}&lastName=${this.$store.state.lastName}&email=${this.$store.state.email}`,
-      )
+    darkDark() {
```

Comment:
consider to implement this in a later PR

Reply:
yes, this is in the next step